GP-Engine-team/GP_Engine
# Changed the scene to load to Game.GPScene

## Description
In MainMenu.GPScene, the "scene to load" is currently set to MainMenu.GPScene.
This pull request changes the scene to load to Game.GPScene

## How to test
Play the game, click "Play" in the menu, and see if the GPGame.GPScene scene is properly loaded
